.travis.yml

```diff
@@ -9,12 +9,13 @@ branches:
     - master
     - /^(edge|stable)-\d+\.\d+\.[\w-]+$/ # build version tags
 
+# TESTING integration tests prior to master merge
 stages:
   - name: test
   - name: docker-deploy
-    if: type != pull_request
+    # if: type != pull_request
   - name: integration-test
-    if: type != pull_request
+    # if: type != pull_request
```

Comment:
note: these changes are to make integration tests run on this PR prior to master merge. will back them out once integration tests are validated.